.gitignore

```diff
@@ -14,7 +14,6 @@
 *.dll
 
 # Fortran module files
-*.mod
```

Comment:
![image](https://user-images.githubusercontent.com/432502/67775927-fdd02c80-fa35-11e9-9125-12418b5d7183.png)

Reply:
ooooooh i get it (for posterity: it looked like we were randomly changing the **Fortran** gitignore and i didnt understand why - but of course, the go module file ends with `.mod` so we need to stop ignoring it. (that must have been an interesting one to track down @wlynch !!)

but i really liked this cat so i dont want to delete this comment... :innocent: